workflow/rules/gene_fuse_report.smk

```diff
@@ -0,0 +1,42 @@
+# vim: syntax=python tabstop=4 expandtab
+# coding: utf-8
+
+__author__ = "Jonas Almlöf"
+__copyright__ = "Copyright 2021, Jonas Almlöf"
```

Comment:
Not super important, but it is 2022 now 😉

Reply:
fixed